polyaxon_schemas/build.py

```diff
@@ -17,7 +17,9 @@
 def validate_image(image):
     if not image:
         raise ValidationError('Invalid docker image `{}`'.format(image))
-    tagged_image = image.split(':')
+    tagged_image = image.rsplit(':')
+    if len(tagged_image) >  3:
+        raise ValidationError('Invalid docker image `{}`'.format(image))
     if len(tagged_image) > 2:
```

Comment:
@mouradmourafiq Do we need this conditional? `> 2` includes `3`, which is valid

Reply:
no I just could not make a suggestion over multiple lines.

Reply:
Okay, no worries